CI/scenarios/ingress_namespace.yaml

```diff
@@ -1,7 +1,7 @@
 scenarios:
-- action: delete
```

Comment:
We might want to change the name of the file to something along the lines of pod scenario. Thoughts?

Reply:
Sorry I don't understand, this scenario actually deletes the namespace, can you give me an example?